operator/metrics/metrics.go

```diff
@@ -93,6 +93,12 @@ var (
 
 	// IdentityGCRuns records how many times identity GC has run
 	IdentityGCRuns *prometheus.GaugeVec
+
+	// CiliumEndpointBatchDensity indicates the total number of CEPs batched in a CEB
+	CiliumEndpointBatchDensity *prometheus.HistogramVec
```

Comment:
Do you actually want a counter here? If you're after the total number a histogram will only give you an approximation of this I believe as its going to place the observed count into a bucket.

Reply:
Actually, we are interested in getting approximate number of CEPs in various buckets. 

For example, 
# of CEBs in <0-10> CEP counts
# of CEBs in <11-20> CEP counts
# of CEBs in <21-30> CEP counts
...
# of CEBs in <91-100> CEP counts